vignettes/what-to-avoid.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "What to avoid"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{what-to-avoid}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, include = FALSE}
+library(dfeshiny)
+```
+
+The R ecosystem provides a wide range of options to create different elements in dashboard type 
+apps in R-Shiny. Whilst this range of choice can be great for creating anything you want in any 
+style you favour, it also leads to a wide range of pitfalls that can undermine your work.
+
+This document outlines a range of packages and elements to avoid and the reasons why, but the key
+underlying considerations around many of these are:
+
+- Continuity: 
+  - Are functions and packages expected to have long term support from their developers?
```

Comment:
Extra points here, which are the main things I tend to check
 - has the package been recently updated?
 - (if code is visible on GitHub) are the issues and pull requests raised on the repository getting responded to quickly?

Reply:
I guess also - if it relies on other packages, is it using the latest version of those (e.g. charting libraries that are a wrapper on JS code, has it been kept up to date with the dependencies updates and major changes - shinyGovstyle and the GDS design system are a good example here too)